language/stdlib/modules/libra_system.mvir

```diff
@@ -24,7 +24,18 @@ module LibraSystem {
 
     resource BlockMetadata {
       // Height of the current block
+      // TODO: Should we keep the height?
       height: u64,
+
+      // current real world time provided by consensus.
```

Comment:
We should be precise about this--isn't it the time that the last block was committed, not the current time?